slick.grid.js

```diff
@@ -1642,60 +1685,71 @@ if (typeof Slick === "undefined") {
    		return impactedColumns;
     }
 
+    function handleResizeableHandleDoubleClick(evt) {
+      const triggeredByColumn = evt.target.parentElement.id.replace(uid, "");
+      trigger(self.onColumnsResizeDblClick, { triggeredByColumn: triggeredByColumn.getReturnValue() });
```

Comment:
this line throws an error because `const triggeredByColumn` is a string and does not have a `.getReturnValue()` function and it work as expected when triggering without it like so `{ triggeredByColumn: triggeredByColumn }`